### PR TITLE
projects/data: Add a new project

### PR DESCRIPTION
**Pull request by arnab1896**
Make changes to data/projects.js and add a new file in
data/projects ---> projects_gsoc.md

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by satwikkansal** [comment]
Please add a short description about the project in commit body and remove 
```
Make changes to data/projects.js and add a new file in
data/projects ---> projects_gsoc.md
````

File: data/projects/projects_gsoc.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* Implemented at least half of the points listed in the proposal.
+* The written code has been tested and documented.
+
+##### CODING PHASE 2
+
+* Implemented, documented and tested second half of points listed.
+
+##### CODING PHASE 3
+
+* Implemented, documented and tested all improvements.
```
Comment (sims1253) [inline data/projects/projects_gsoc.md]: second half leads to 100% being done in phase two, so nothing to do here

File: data/projects/projects_gsoc.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* This will be a buffer period for any remaining points and implementing
```
Comment (sims1253) [inline data/projects/projects_gsoc.md]: Buffer period is no milestone ;)

File: data/projects.js [hunk 3]
```diff
@@ -447,6 +447,24 @@ coala should support generating metrics for your code.",
...
+		"name" : "Build an ideas website for GSoC",
+		"desc" : "There are a number of possible ways to improve the \
+                          implementation of GSoC keeping [projects](https://projects.coala.io) \
+                          in mind.",
+		"requirements" : [
+			"One patch to projects accepted.",
+			"Clear ideas on how to improve the GSoC ideas implementation \
+                         in the proposal."
+		],
+		"difficulty" : "medium",
```
Comment (sims1253) [inline data/projects.js]: Although I am lacking experience with web dev stuff I feel like this deserves an easy compared with our other projects

**Comment by jayvdb** [comment]
There are enough pending issues with this proposal, and now isnt the time to be refining this proposal, so I am closing this PR and I've opened https://github.com/coala/projects/issues/394 where the discussion around this project concept can continue.

**Pull request closed without merging**